File: python/design-patterns/abstractfactory/naive/screen_naive.py
def render_login_screen(os):
    # ❌ one dev writes the button branch…
    button = MacButton() if os == "mac" else WindowsButton()

    # ❌ …another writes the checkbox branch, defaulting to Mac:
    checkbox = WindowsCheckbox() if os == "windows" else MacCheckbox()

    # 💥 On Windows: Windows button next to a Mac checkbox.
    return [button.render(), checkbox.render()]

File: java/design-patterns/abstractfactory/naive/ScreenNaive.java
List<String> renderLoginScreen(String os) {
    // ❌ one dev writes the button branch…
    Button button = os.equals("mac") ? new MacButton() : new WindowsButton();

    // ❌ …another writes the checkbox branch, defaulting to Mac:
    Checkbox checkbox = os.equals("windows")
        ? new WindowsCheckbox() : new MacCheckbox();

    // 💥 On Windows: Windows button next to a Mac checkbox.
    return List.of(button.render(), checkbox.render());
}